.travis.yml

```diff
@@ -1,14 +1,16 @@
 language: php
 php:
   - 5.6
+  - 7.0
```

Comment:
Would be nice to add `7.1`, `7.2`, and `nightly` (this last one as `allowed_failure`)

Reply:
7.1 was running into `composer and ubuntu` issues, so i left it out for @bracki to evaluate